geonetwork/transform-mcp-iso19115-3/schema/iso19115-3/process/transform-19115-3-testing-urls.xsl

```diff
@@ -48,13 +51,21 @@
 
   <xsl:template mode="substitute" match="@*|text()">
     <xsl:variable name="url" select="."/>
- 
     <xsl:for-each select="$urlSubstitutions/substitution">
       <xsl:if test="matches($url, string(@match))">
         <xsl:value-of select="replace($url, string(@match), string(@replaceWith))"/>
       </xsl:if>
     </xsl:for-each>
   </xsl:template>
 
+  <xsl:template match="mri:abstract/gco:CharacterString">
+    <xsl:variable name="abstractText" select="text()"/>
+    <xsl:for-each select="$urlSubstitutions/substitution">
```

Comment:
Need to use an xsl:copy or an xsl:element element to create the gco:CharacterString element we've matched agaianst but not written to the output here yet.  Currently this code is removing the gco:CharacterString element.

Reply:
Whoops! I didn't notice that... it should be all fixed up now.